README.md

```diff
@@ -127,12 +127,12 @@ python3 convert.py json | jq -r '(.Analytics,.Doubleclick) | .[]'
 ## Can I block the other letters of GAFAM ?
 Of course, here’s some filterlists that should help you accomplish that.
 
-**A**pple : https://github.com/c-edw/ios-telemetry or https://github.com/1r2/iosparanoid
+**A**pple : https://github.com/c-edw/ios-telemetry , https://github.com/1r2/iosparanoid or https://raw.githubusercontent.com/hagezi/dns-blocklists/main/hosts/native.apple.txt
```

Comment:
```suggestion
**A**pple : https://github.com/c-edw/ios-telemetry, https://github.com/1r2/iosparanoid or https://raw.githubusercontent.com/hagezi/dns-blocklists/main/hosts/native.apple.txt
```

Reply:
Change made, if something is wrong do not hesitate to let me know.